infra/modules/providers/azure/app-service/main.tf

```diff
@@ -58,10 +58,9 @@ resource "azurerm_app_service" "appsvc" {
   )
 
   site_config {
-    linux_fx_version     = local.app_linux_fx_versions[local.app_names[count.index]]
-    always_on            = var.site_config_always_on
-    virtual_network_name = var.vnet_name
```

Comment:
This is actually a breaking change. It does look like its not supported anymore in the new TF provider, but I wonder if there is a different way that we can configure it?

The impact is that these templates are no longer VNET isolated.

Reply:
We sill have a VNET isolated app-service by using **resource "azurerm_template_deployment"** we are using [here](https://github.com/microsoft/cobalt/blob/master/infra/modules/providers/azure/app-service/main.tf#L135)